replication project.Rmd

```diff
@@ -0,0 +1,259 @@
+
+#The paper
+Kai Arzheimer, "Contextual Factors and the Extreme Right Vote in Western Europe, 1980-2002"25548117.pdf
+This article attempts to answer the “twin question of why the extreme right support is so unstable within many countries over time, and why these parties are so weak in many West European countries.” The author conduct a multi-level analysis, combining data on macro-level contextual variables (unemployment, immigration, and welfare benefits) with individual-level variables (sociodemographic factors and attitudes). It concludes that although the contextual factors are positively correlated with the intention to vote for an extreme-right party, they don't seem to reinforce each other (i.e. as unemployment levels rise, the effect of immigration does not increase but rather declines).
+
+#The data
+Individual-level variables come from Eurobarometer surveys (1980-2002), while contextual variables are gathered from a number of different data sets produced by the OECD, UNHCR etc. The merged data and the Stata code are available through the author’s dataverse at: http://hdl.handle.net/1902.1/12092. 
+
+#The model 
+The article uses a multi-level logistic model via Penalized Quasi-Likelihood. The dependent variable is vote intention for an extreme right party, which is assumed to be binomially distributed.
+
+# Run necessary packages and load data (Source: Arzheimer dataverse)
+
+```{r}
+
+
+library(MASS)
+library(nlme)
+library(xtable)
+library(foreign)
+library(dplyr)
+library(broom)
+library(ggplot2)
+library(xtable)
+library(stargazer)
+library(grid)
+library(gridExtra)
+
+data <- read.dta("nonimp.dta")
+
+# Cleaning up data
+data <- rename(data, ext_vote = rexvote)
+data <- rename(data, age_1 = age1) # (18–29 years)
+data <- rename(data, age_2 = age2) # (30–45 years)
+data <- rename(data, age_4 = age4) # (>65 years)
+data <- rename(data, edu_1 = mye1) # Education: middle/high
+data <- rename(data, edu_2 = mye2) # Education: university
+data <- rename(data, farmer_own = farmerown) 
+data <- rename(data, left_right_scale = zlrs) # Left-Right Scale
+data <- rename(data, eu_neg = euschlecht) # Negative evaluation of EU membership of one's own country
+data <- rename(data, z_dem_satis = zsatisdmo) # Dissatisfied: Democracy
+data <- rename(data, dispro_elect = disp) # Disproportionality
+data <- rename(data, federalism = lfed1) # Decentralization
+data <- rename(data, z_asylumseeker = zasylumseekers) 
+data <- rename(data, z_unemp = zsur) # Unemployment rate
+data <- rename(data, z_replacement = zreplacementrate) # Unemp benefits
+data <- rename(data, max_er = rmax) # Maximum toughness in extreme right discourse
+
+```
+
+
+# Preparing data for analysis
+
+```{r}
+# Create centered variables for Saliance and Variance based on grand mean centering according to author
+
+data <- data  %>% 
+  mutate(salience_mean_c = (salienzmean - 3.84568)) %>% 
+  mutate(var_c = rvar - 21.75423)
+
```

Comment:
why hard code 3.84568? why not calculate that?  That would be easier to read and less prone to error.